### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by RavanaPiriyeva**

### CODE REVIEW

File: src/pages/CartPage/CartPage.scss [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell',
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+    line-height: 1.5;
+    color: #102a42;
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
```
Comment (Negodyaj) [inline src/pages/CartPage/CartPage.scss:16]: first 16 lines must be in index.scss

File: src/pages/CartPage/CartPage.scss [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+  h3 {
+      font-size: 2rem ;
+  }
+}
+
+@media screen and (min-width: 800px){
+h4 {
+    font-size: 1.5rem;
+}
+}
```
Comment (Negodyaj) [inline src/pages/CartPage/CartPage.scss:60]: all those styles will be applied for all the pages, not only Cart. is it expected?
or all those styles for cart page only?

File: src/pages/CartPage/CartPage.scss [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+    gap: 1rem;
+    text-align: left;
+    }
+    }
+    @media (min-width:776px){
+            grid-template-columns: 1fr 1fr 1fr 1fr auto;
+    align-items: center;
+    grid-template-rows: 75px;
+    }
+}
```
Comment (Negodyaj) [inline src/pages/CartPage/CartPage.scss:370]: please, use Alt+Shift+F from time to time...)))

File: src/pages/CartPage/CartPage.tsx [hunk 4]
```diff
@@ -1,7 +1,110 @@
...
+          <h5>Subtotal</h5>
+          <span></span>
+        </div>
+        <hr />
+      </div>
+
+      <div className="product">
+        <div className="title">
+          <img
+            src="https://v5.airtableusercontent.com/v1/13/13/1671710400000/cBADR3Ykxoxl0MkXX9kQPQ/bJszrcHzCdSytgEFhJtanrKnPaDEew9L11QHw58ggJfBzzCbeIYljmDX3fYaey80HY80yiZ3DzOyZ12oCgq55mTas8tv_kdRegN9uvyMGVGn90rXjfbfef-2V3uzZ5p8/A8ZFbRYnMhjnWkcww_lRxdja-IXBQFG1S0aU-cEg39M"
```
Comment (Negodyaj) [inline src/pages/CartPage/CartPage.tsx:21]: I think it's better to download this image and put it next to your CartPage.tsx

File: src/pages/CartPage/CartPage.scss (hunk 2 above)
Comment (RavanaPiriyeva) [reply src/pages/CartPage/CartPage.scss:60]: yes

File: src/pages/CartPage/CartPage.scss (hunk 1 above)
Comment (RavanaPiriyeva) [reply src/pages/CartPage/CartPage.scss:16]: but this change all page

**Pull request merged**